src/api/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.use('/v1', require('./v1'));
```

Comment:
We should use ES modules imports

Reply:
Yes. We'll be using ES modules on this project. I'm sure it was an oversight as the commonJS syntax came with the template.